src/content/configuration/entry-context.mdx

```diff
@@ -185,3 +186,25 @@ module.exports = {
 ```
 
 When combining with the [`output.library`](/configuration/output/#outputlibrary) option: If an array is passed only the last item is exported.
+
+### Runtime chunk
+
+It allows to set the runtime chunk for an entrypoint and it can be set to `false` to avoid a new runtime chunk since webpack 5.43.0.
```

Comment:
```suggestion
It allows setting the runtime chunk for an entry point and setting it to `false` to avoid a new runtime chunk since webpack `v5.43.0`.
```